### PR TITLE
Rolled back head request check and catching

### PR DESCRIPTION
**Pull request by rodrigopavezi**
status 0 as error

@stulees @donnapep please review. cheers

### CODE REVIEW

File: src/js/rise-cache.js [hunk 1]
```diff
@@ -49,51 +49,53 @@ RiseVision.Common.RiseCache = (function () {
...
+          xhr: xhr,
+          url: url
+        };
+
+        xhr.open("GET", url, true);
+
+        xhr.addEventListener('loadend', function () {
+          var status = xhr.status || 0;
+
+          if (status === 0 || (status >= 200 && status < 300)) {
```
Comment (stulees) [inline src/js/rise-cache.js]: @rodrigopavezi I think you need to remove the `status === 0` as this was a fix put in place prior to the HEAD request functionality. See this previous commit here - https://goo.gl/7yKgXC

Comment (stulees) [inline src/js/rise-cache.js]: @rodrigopavezi Or is this what is also causing an issue?

Comment (donnapep) [inline src/js/rise-cache.js]: @stulees @rodrigopavezi That should stay. It's unrelated to the functionality that is being rolled bakck.

**Comment by donnapep** [comment]
:+1:

File: src/js/rise-cache.js (hunk 1 above)
Comment (rodrigopavezi) [inline src/js/rise-cache.js]: @stulees @donnapep As I understood If there is a cross domain issue the status will be 0 so we have to consider it. cheers

Comment (stulees) [inline src/js/rise-cache.js]: @rodrigopavezi Okay, thanks for clarifying.

**Comment by stulees** [comment]
@rodrigopavezi :+1:

**Pull request closed without merging**